### PR TITLE
MCON-11: Disable participants invite for inactive conferences

### PR DESCRIPTION
**Pull request by azefiel**

### CODE REVIEW

File: src/apps/common/submodules/conferenceViewer/conferenceViewer.js [hunk 1]
```diff
@@ -246,67 +246,71 @@ define(function(require) {
...
-			formattedData.conference.disabledActions = formattedData.conference.participants.length + formattedData.conference.moderators.length === 0;
-
-			return formattedData;
+				users = self.conferenceViewerFormatUsers(data.participants),
+				moderators = _.filter(users, { is_moderator: true }),
+				participants = _.filter(users, { is_moderator: false }),
+				hasUsers = _.some([
+					moderators,
+					participants
+				], _.negate(_.isEmpty));
```
Comment (guillegr123) [inline src/apps/common/submodules/conferenceViewer/conferenceViewer.js:255]: Is it possible to only check if the participants list is not empty?
```suggestion
				hasUsers = _.some(users);
```

Comment (azefiel) [reply src/apps/common/submodules/conferenceViewer/conferenceViewer.js:255]: Good catch, checking `users`'s emptiness would indeed be more straightforward!

**Comment by azefiel** [comment]
Also updated the moderator/participants resolver to exactly match previous behavior (as not certain non-moderator necessarily have an `is_moderator` prop).

File: src/apps/common/submodules/conferenceViewer/conferenceViewer.js [hunk 2]
```diff
@@ -246,67 +246,72 @@ define(function(require) {
...
+				),
+				users = self.conferenceViewerFormatUsers(data.participants),
+				hasUsers = !_.isEmpty(users),
+				moderators = _.filter(users, isModerator),
+				participants = _.reject(users, isModerator);
+
+			return {
+				backButton: args.backButton,
+				conference: _.merge({
+					disabledActions: !hasUsers,
```
Comment (guillegr123) [inline src/apps/common/submodules/conferenceViewer/conferenceViewer.js:261]: This value does not seem to be read anywhere within this repository. Will it be used in the future?

Comment (azefiel) [reply src/apps/common/submodules/conferenceViewer/conferenceViewer.js:261]: After do [looking into it](https://github.com/2600hz/monster-ui/commit/572e49cfc61898c5ecc81a1c95054dbfb104708f), it does seem like it being used.

**Review by guillegr123 — APPROVED** [review]
Looks good to me!

**Pull request merged**